mlflow/sklearn/__init__.py

```diff
@@ -1146,6 +1159,41 @@ def fetch_logged_data(run_id):
                                       ``True``. See the `post training metrics`_ section for more
                                       details.
     """
+    _autolog(
+        flavor_name=FLAVOR_NAME,
+        log_input_examples=log_input_examples,
+        log_model_signatures=log_model_signatures,
+        log_models=log_models,
+        disable=disable,
+        exclusive=exclusive,
+        disable_for_unsupported_versions=disable_for_unsupported_versions,
+        silent=silent,
+        max_tuning_runs=max_tuning_runs,
+        log_post_training_metrics=log_post_training_metrics,
+        xgboost_estimator=False,
+    )
+
+
+def _autolog(
+    flavor_name=FLAVOR_NAME,
```

Comment:
Internal API for sklearn autologging. The `flavor_name` field allows `mlflow.xgboost` to specify the `xgboost_sklearn` flavor, preventing flavor conflict with `mlflow.sklearn`.